script/actions_utils/update_slab_github_runner.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
I added this file to automatically update the SHA.

Initially, I changed `zama-ai/action/workflows@...` (which has 18 occurrences). Based on that, I thought it would be useful to add a script to handle this automatically.

But, I later realized that the correct reference to update is `zama-ai/slab-github-runner@....`

Would you like to keep the script, or should we remove it? @andrei-stoian-zama?

Reply:
We agreed to remove it.